src/ui/components/Rating/index.js

```diff
@@ -46,6 +46,16 @@ export class RatingBase extends React.Component {
     this.props.onSelectRating(rating);
   }
 
+  checkStarRating = (rating, compareRating) => {
+    const roundRating = parseFloat(rating.toFixed(1));
+    if (roundRating - compareRating >= 0) {
+      return 1;
+    } else if (((roundRating + 0.5) - compareRating) >= 0) {
```

Comment:
This logic is sort of confusing... I get what it's doing but it would be clearer if the code expressed the intention better, eg:

```js
if (roundRating + 0.5 - compareRating < 0) {
  return 0; // We only return zero if the rating average is really bad!
}

return 1;
```